### DISCUSSION TITLE
Unable to build

### DISCUSSION THREAD

**Discussion by briancoates** [mxstbr/karabiner#6] [Q&A] [answered]
```
❯ yarn run build
yarn run v1.22.19
$ tsm rules.ts
/bin/sh: tsm: command not found
error Command failed with exit code 127.
```

I've installed yarn with both homebrew and npm, neither resolves the issue.  Checked documenation, can't figure out what `tsm`.

I'm new to typescript and yarn, so I'm assuming its something with me.

**Comment by briancoates** [comment, marked as answer]
Found it.  For anyone else...

https://github.com/lukeed/tsm

**Reply by mxstbr** [reply]
Glad you figured it out! Will add to the dependencies so it works out of the box.
